packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -19,79 +17,43 @@ import {
   TableHead,
   TableRow,
   TableContainer,
-  TablePagination,
   TableSortLabel,
   Table as MuiTable,
 } from '@material-ui/core';
 
 import { SortDesc } from '../../icons';
-import { DEFAULT_PAGE_SIZE } from '.';
 import { TableProps } from './types';
 import { useSetupDataTableApi } from './hooks/useDataTableApi';
 import { DataRow } from './components/DataRow/DataRow';
+import { PaginationRow } from './columns/PaginationRow';
 
 export { SortingRule };
 
 export const RemoteDataTable = <T extends Record<string, unknown>>({
   columns,
+  sortBy,
   data = [],
-  initialSortBy,
   isLoading = false,
-  onFetchData,
+  onSortBy,
   onRowClick,
-  totalLength = 0,
+  pagination,
   tableApi,
+  onChangePage,
```

Comment:
I just pass the `sortBy` instead of `initialSortBy`. We were keeping the current `sortBy` in the `ListView` anyway. Then just needed a `onSortBy` to remove the `useSortBy` hook which doesn't really isn't doing anything..

Then .. (I actually implemented this by re-fetch the data by the number of rows to render rather than just slicing the data [ uhh.. 🙄]) I extracted all the query props into the one hook which could be a bit tidier but will do with the `useListData` hook later